osbs/http.py

```diff
@@ -34,6 +39,21 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
```

Comment:
I thought the whole point of subclassing `Retry` was so we could hook some logging. This is not being done anymore.
Why not just use `status_forcelist` ?